src/components/Common/ContextMenu.tsx

```diff
@@ -151,7 +151,7 @@ export default function ContextMenu({
         ) : (
           <ContextMenuItemRenderer {...props} highlightColor={highlightColor} />
         );
-        return <Fragment key={props.name}>{Renderer}</Fragment>;
+        return <Fragment key={props.dataQa}>{Renderer}</Fragment>;
```

Comment:
`dataQA` is required property and I use it because it must be unique in comparison with `name` i.e. we have two `MoveTo` items (for desktop and for mobile)